NitroxServer/Server.cs

```diff
@@ -13,21 +15,22 @@ public class Server
         private readonly WorldPersistence worldPersistence;
         private readonly PacketHandler packetHandler;
         private readonly Timer saveTimer;
-
+        private ServerConfigReader ServerOptions;
         public static Server Instance;
 
         public void Save()
         {
             worldPersistence.Save(world);
         }
 
-        public Server()
+        public Server(ServerConfigReader ConfigReader)
```

Comment:
parameters for methods/functions are camelCase. `ConfigReader` to `configReader`.